### ISSUE TITLE
Create how to review a PR doc

### ISSUE THREAD

**Issue by FedericoEsparza** [AgileVentures/sing_for_needs#65]
Create how to review a PR doc

In a new folder, `devdocs`, replicate the following doc for this repo:
https://github.com/championer-org/championer_one/blob/develop/devdocs/HOW-TO-REVIEW-A-DEVELOP-PR.md

**Make sure the links don't point to the `championer_one` repo doc.**

It's not a certainty that we'll be using Hound and Chromedriver for this project, so, for now, remove the references to it, and simplify the test command to:
```
mix test
```
The tl;dr commands are useful for speeding up PR reviews, so it's nice to have them handy in the repo.
